src/Symfony/Component/Console/Tests/Fixtures/application_renderexception4.txt

```diff
@@ -1,9 +1,9 @@
 
 
-                               
-  [InvalidArgumentException]   
-  Command "foo" is not define  
-  d.                           
-                               
+                                                                    
+  [Symfony\Component\Console\Exception\CommandNotDefinedException]  
+  Command "foo" is not define                                       
+  d.                                                                
```

Comment:
I'm not sure this is the correct behavior. The line is wrapped, but not the class name, and the rest of the line is filled with spaces.